plugins/auth-backend/config.d.ts

```diff
@@ -53,6 +53,7 @@ export interface Config {
         decryptionPvk?: string;
         signatureAlgorithm?: 'sha256' | 'sha512';
         digestAlgorithm?: string;
+        acceptedClockSkewMs: number;
```

Comment:
This should be `acceptedClockSkewMs?` to make it optional.

Reply:
I fixed it